### PR TITLE
Support Laravel 7

### PR DESCRIPTION
**Pull request by wilburpowery**

### CODE REVIEW

**Comment by rasmuscnielsen** [comment]
Hey Wilbur

Thanks for this PR! Could you also update the Travis config to test Laravel 7 please? :-) 

Cheers

**Comment by wilburpowery** [comment]
Sure, no problem :)

**Comment by rasmuscnielsen** [comment]
@wilburpowery Just made some tweaks to make everything pass and test properly.

The final thing I'd ask you to do is if could do a git rebase to a single commit which looks a bit nicer? Otherwise I can do a squash merge, but I'd prefer to make sure your contribution shows up in the history and gets accredited properly with you as author :-)

In case you haven't done it before, you can find some information here:
https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d
https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

Basically you want to do something like git rebase -i HEAD~10 and then squash all subsequent commits into the first "Bump to Laravel 7" + and then force push :-)
The articles should explain it fairly well 👍

**Comment by wilburpowery** [comment]
Thanks @rasmuscnielsen! Done 🤠
